### PR TITLE
Div 5457 update image to latest

### PR DESCRIPTION
**Pull request by jacmurphy**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DIV-5457

### Change description ###
Add`image: hmctspublic.azurecr.io/div/dn:latest` to both AAT and Demo

### CODE REVIEW

**Review by Dizrupto — APPROVED** [review]

File: k8s/aat/common/divorce/div-dn.yaml [hunk 1]
```diff
@@ -20,7 +20,7 @@ spec:
     nodejs:
       replicas: 2
       useInterpodAntiAffinity: true
-      image: hmctspublic.azurecr.io/div/dn:prod-947a296f
+      image: hmctspublic.azurecr.io/div/dn:latest
```
Comment (Dizrupto) [inline k8s/aat/common/divorce/div-dn.yaml:23]: Did someone confirm this is okay? :)

Comment (madjava) [reply k8s/aat/common/divorce/div-dn.yaml:23]: @jacmurphy Looking at the commit history, the :latest tag has never been used. Might want to verify this

Comment (jacmurphy) [reply k8s/aat/common/divorce/div-dn.yaml:23]: It was the solution that Dale was given by Luigi - https://hmcts-reform.slack.com/archives/C8SR5CAMU/p1616592447229600?thread_ts=1616590641.226700&cid=C8SR5CAMU

**Review by madjava — CHANGES_REQUESTED** [review]
Please confirm image tag

File: k8s/aat/common/divorce/div-dn.yaml (hunk 1 above)
Comment (madjava) [reply k8s/aat/common/divorce/div-dn.yaml:23]: Ok, its this the way to go for this then

**Pull request closed without merging**